src/controllers/__tests__/organizationController.test.ts

```diff
@@ -393,27 +450,112 @@ describe("OrganizationController", () => {
   });
 });
 
-describe("OrganizationController#addDelegate()", () => {
-  const alreadyExistingDelegates: ReadonlyArray<OrganizationDelegate> = [
-    mockedDelegate1,
-    mockedDelegate2
-  ];
-  const registeredOrganization: Organization = {
-    ...mockedRegisteredOrganization1,
-    users: alreadyExistingDelegates
-  };
-  const delegateToBeAdded: OrganizationDelegate = {
-    email: mockedLoggedDelegate.email,
-    family_name: mockedLoggedDelegate.familyName,
-    fiscal_code: mockedLoggedDelegate.fiscalCode,
-    given_name: mockedLoggedDelegate.givenName,
-    role: mockedLoggedDelegate.role,
-    work_email: mockedLoggedDelegate.workEmail
-  };
-  const registeredOrganizationWithAddedDelegate = {
-    ...registeredOrganization,
-    users: alreadyExistingDelegates.concat(delegateToBeAdded)
-  };
+describe("OrganizationController#sendDocuments()", () => {
+  // tslint:disable-next-line:no-let
+  let organizationRegistrationRequestModel: RequestModel;
+  // tslint:disable-next-line:no-let
+  let userDelegationRequestModel: RequestModel;
+  // tslint:disable-next-line:no-let
+  let requestModelFromOtherUser: RequestModel;
+  // tslint:disable-next-line:no-let
+  let requestModelForAnotherAdministration: RequestModel;
+  // tslint:disable-next-line:no-let
+  let requestModelAlreadySubmitted: RequestModel;
+  // tslint:disable-next-line:no-let
+  let loggedDelegateModel: UserModel;
+  // tslint:disable-next-line:no-let
+  let otherUserModel: UserModel;
+  beforeAll(async () => {
```

Comment:
This is overkill  :) Does this code create then destroys things in the real database ? We must rewrite this part to just have the calls to DB methods mocked using jest.